.github/release-drafter.yml

```
@@ -0,0 +1,51 @@
+name-template: "$RESOLVED_VERSION"
+tag-template: "$RESOLVED_VERSION"
+
+categories:
+  - title: "Backwards Incompatible Changes"
+    labels:
+      - "backwards incompatible"
+
+  - title: "Features"
+    labels:
+      - "feature"
+      - "enhancement"
+      # usually combined with enhancement (avoid double listing)
+      - "deprecation"
+
+  - title: "Bug Fixes"
+    labels:
+      - "bug"
+      - "bug fix"
+
+  - title: "Other Changes"
+    labels:
+      - "dependency"
+      - "docs"
+      - "trivial"
+      - "maintenance"
+
+change-template: "- $TITLE (#$NUMBER) @$AUTHOR"
```

Current template is:

```suggestion
change-template: "- $TITLE ([#$NUMBER](https://github.com/jazzband/pip-tools/pull/$NUMBER)). Thanks @$AUTHOR"
```